infrastructure/stacks/service_etl/functions/service_etl/service_etl.py

```diff
@@ -109,7 +109,9 @@ def extract_data_from_dos():
                             array_to_string(array(select endtime from pathwaysdos.servicedayopenings sdo inner join pathwaysdos.openingtimedays o  on sdo.dayid = o.id inner join pathwaysdos.servicedayopeningtimes sdot on sdo.id = sdot.servicedayopeningid where s.id = sdo.serviceid order by sdo.dayid), ',') as closingtime,
                             array_to_string(array(select date from pathwaysdos.servicespecifiedopeningdates sod where sod.serviceid = s.id),',') as specifieddates,
                             array_to_string(array(select starttime from pathwaysdos.servicespecifiedopeningtimes sot inner join pathwaysdos.servicespecifiedopeningdates spcd on sot.servicespecifiedopeningdateid = spcd.id where spcd.serviceid = s.id),',') as specificopentimes,
-                            array_to_string(array(select endtime from pathwaysdos.servicespecifiedopeningtimes sot inner join pathwaysdos.servicespecifiedopeningdates spcd on sot.servicespecifiedopeningdateid = spcd.id where spcd.serviceid = s.id),',') as specificendtimes
+                            array_to_string(array(select endtime from pathwaysdos.servicespecifiedopeningtimes sot inner join pathwaysdos.servicespecifiedopeningdates spcd on sot.servicespecifiedopeningdateid = spcd.id where spcd.serviceid = s.id),',') as specificendtimes,
+                            s.latitude,
+                            s.longitude
                         from
                             pathwaysdos.services s,
                             pathwaysdos.servicecapacities sc,
```

Comment:
were you able to find a solution to the professional referral filter

Reply:
added where condition in the query